common/spec/dependabot/config/ignore_condition_spec.rb

```diff
@@ -207,7 +276,14 @@ def expect_ignored(versions)
         context "with ignore_minor_versions" do
           let(:update_types) { ["version-update:semver-minor"] }
           it "returns the expected range" do
-            expect(ignored_versions).to eq([">= Finchley.a, < Finchley.999999"])
+            expect(ignored_versions).to eq([])
```

Comment:
Aside: this is a regression from how `@dependabot ignore` behaves on such a request.
I'm cool with that, as it seems consistent for the `semver-*` to enforce semver semantics and not sweat the edge cases.